src/libPMacc/include/memory/buffers/DeviceBufferIntern.hpp

```diff
@@ -56,14 +56,14 @@ class DeviceBufferIntern : public DeviceBuffer<TYPE, DIM>
         //create size on device before any use of setCurrentSize
         if (useVectorAsBase)
         {
-            sizeOnDevice = false;
-            createSizeOnDevice(sizeOnDevice);
+            this->sizeOnDevice = false;
+            createSizeOnDevice(this->sizeOnDevice);
             createFakeData();
             this->data1D = true;
         }
         else
         {
-            createSizeOnDevice(sizeOnDevice);
```

Comment:
The parameter `sizeOnDevice` is used in [line 52](https://github.com/ComputationalRadiationPhysics/picongpu/pull/2051/files/cb08b91920612dd15dc5695d0b407eae832608aa#diff-b55f61e3b7f4d0536bd72930ba24ad25L52).